From 8eaa18defface5e20f0af454edc2b79d925af05e Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 12:21:17 -0700
Subject: [PATCH] Docs/translations/ko_KR: move howto.rst under process/
 directory

The menu for documents of other available languages comes up only if the
documents are in same file hierarchy.  Because howto.rst of Korean
translation is at the root while that for English is under howto/
directory, the Korean translation is not listed on the available
versions menu of the original document and other translations.  Move the
document under howto/ directory to make it linked with other language
versions.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/translations/ko_KR/index.rst               | 2 +-
 Documentation/translations/ko_KR/{ => process}/howto.rst | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename Documentation/translations/ko_KR/{ => process}/howto.rst (100%)

diff --git a/Documentation/translations/ko_KR/index.rst b/Documentation/translations/ko_KR/index.rst
index 4add6b2fe1f2..2d51f1481310 100644
--- a/Documentation/translations/ko_KR/index.rst
+++ b/Documentation/translations/ko_KR/index.rst
@@ -11,7 +11,7 @@
 .. toctree::
    :maxdepth: 1
 
-   howto
+   process/howto
 
 
 리눅스 커널 메모리 배리어
diff --git a/Documentation/translations/ko_KR/howto.rst b/Documentation/translations/ko_KR/process/howto.rst
similarity index 100%
rename from Documentation/translations/ko_KR/howto.rst
rename to Documentation/translations/ko_KR/process/howto.rst
-- 
2.39.2

